root/frontend/src/App.jsx

```diff
@@ -55,32 +55,137 @@ class App extends Component {
       showAlert: ['success', 'Semester added to flowchart. Drag any classes into the new semester.'], 
       Semesters: [...this.state.Semesters, semester] 
     });
+    /*/ 
+      This is where the "Add Semseter" API call exists 
+    /*/
+
+    // Use the split() function to split the string into an array
+    // based on the '-' delimiter
+    var semesterArray = semester.split('-');
+    // Store the first element of the array (i.e., "Summer") as lowercase into a variable
+    var season = semesterArray[0].toLowerCase();
+    // Store the second element of the array (i.e., "1") into a variable and cast it to an integer
+    var number = parseInt(semesterArray[1]);
+
+    console.log(season)
+    console.log("\n")
+    console.log(number)
+    console.log("\n")
```

Comment:
Is there a reason why you put all these console.logs onto multiple lines? I am okay with it, I just wonder if it will clutter the readability